src/components/screens/sprint-view-screen.tsx

```diff
@@ -1,11 +1,270 @@
+
+  /**
+   * Get project name
+   */
+  const getProjectName = (allocation: Allocations) => {
+    if (projects.length !== 0) {
+      return projects[allocations.indexOf(allocation)]?.name;
+    }
+    return "";
+  }
+
+  /**
+   * Get project color
+   */
+  const getProjectColor = (allocation: Allocations) => {
+    if (projects.length !== 0) {
+      return projects[allocations.indexOf(allocation)]?.color;
+    }
+    return "";
+  }
+
+  /**
+   * Calculate total unallocated time for the user 
+   */
+  const unallocatedTime = (allocation: Allocations[]) => {
+      const totalAllocatedTime = allocation.reduce((total, allocation) => total + totalAllocations(allocation), 0);
+      const calculateWorkingLoad = (person?: Person) => {
+        if (!person) {
+          return 0;
+        }
+        const totalMinutes =
+          (person.monday || 0) + 
+          (person.tuesday || 0) + 
+          (person.wednesday || 0) + 
+          (person.thursday || 0) + 
+          (person.friday || 0);
+        return totalMinutes * 2;
+      }
+      return calculateWorkingLoad(loggedInPerson) - totalAllocatedTime;
+    }
+
+  /**
+   * Featute for task filtering 
+   */
+  const handleOnClickTask = () => {
+    setMyTasks(!myTasks);
+    setFilter("");
+  }
+  
   return (
-    <h1>{ strings.sprint.sprintviewScreen }</h1>
+    <>
+      {loading ? (
+        <Card sx={{ p: "25%", display: "flex", justifyContent: "center" }}>
+          <Box sx={{textAlign: 'center'}} >
+            <Typography>{strings.placeHolder.pleaseWait}</Typography>
+            <CircularProgress sx={{ scale: "150%", mt: "5%", mb: "5%"}} />
+          </Box>
+        </Card>
+      ) : (
+        <>
+          <FormControlLabel control={<Switch checked={myTasks}/>} label={strings.sprint.showMyTasks}  onClick={() =>  handleOnClickTask()}/>  
+          <FormControl size="small"  style= {{width: "200px", float:"right"}}>           
+            <InputLabel disableAnimation={false} >{strings.sprint.taskStatus}</InputLabel>
+            <Select
+              defaultValue={strings.sprint.allTasks}
+              style={{borderRadius: "30px", marginBottom: "15px" , float:  "right"}}
+              label={strings.sprint.taskStatus}             
+            >   
+              <MenuItem key={1} value={strings.sprint.toDo} onClick={() => setFilter("TODO")} >
+                {strings.sprint.toDo} 
+              </MenuItem>
+              <MenuItem key={2} value={strings.sprint.inProgress} onClick={() => setFilter("INPROGRESS")}>
+                {strings.sprint.inProgress}
+              </MenuItem>
+              <MenuItem key={3} value={strings.sprint.allTasks} onClick={() => setFilter("DONE")}>
+                {strings.sprint.completed}
+              </MenuItem>
+              <MenuItem key={4} value={strings.sprint.allTasks} onClick={() => setFilter("")}>
+                {strings.sprint.allTasks}
+              </MenuItem>
+            </Select>
+          </FormControl>
```

Comment:
This should probably be removed for now as it does not work until there are actually statuses to filer by, which is being done in a future branch I believe

Reply:
It does if update spec